frontend/src/IndexForImport.tsx

```diff
@@ -0,0 +1,7 @@
+export {default as Dashboard} from './pages/dashboard/Dashboard';
```

Comment:
Very interesting approach, is that file for pages only, or is it for everything?

Reply:
thanks. it is just used for importing components to app.tsx. to get more cleane codes there.